Frontend/ui-customer/src/Screens/OnBoardingFlow/PermissionScreen/View.purs

```diff
@@ -53,6 +53,7 @@ view triggertype push state =
   [ height MATCH_PARENT
   , width MATCH_PARENT
   , clickable true
+  , visibility if (EHC.os == "IOS") then GONE else VISIBLE
```

Comment:
check the current screen is for location or not. for internet we need this screen.